### PR TITLE
Changes related to wrong size of image in Mac

### PR DESCRIPTION
**Pull request by DaddyWesker**
This PR should be tested with [another PR in minecraft-demo](https://github.com/trueagi-io/minecraft-demo/pull/136). Not separetely.

### CODE REVIEW

File: src/main/java/io/singularitynet/utils/ImageClass.java [hunk 1]
```diff
@@ -0,0 +1,804 @@
...
+import org.lwjgl.stb.STBImageResize;
+import org.lwjgl.stb.STBImageWrite;
+import org.lwjgl.stb.STBTTFontinfo;
+import org.lwjgl.stb.STBTruetype;
+import org.lwjgl.system.MemoryStack;
+import org.lwjgl.system.MemoryUtil;
+import org.slf4j.Logger;
+
+@Environment(value=EnvType.CLIENT)
+public final class ImageClass
```
Comment (noskill) [inline src/main/java/io/singularitynet/utils/ImageClass.java:47]: Could you please add comment where this class comes from

Comment (DaddyWesker) [reply src/main/java/io/singularitynet/utils/ImageClass.java:47]: @noskill  this is a clone of NativeImage. It was cloned to change function writeTo so it now saves bmp not png bytes.

Comment (noskill) [reply src/main/java/io/singularitynet/utils/ImageClass.java:47]: I mean add comment to the source code.

Comment (DaddyWesker) [reply src/main/java/io/singularitynet/utils/ImageClass.java:47]: @noskill  done

**Review by Necr0x0Der — APPROVED** [review]

**Review by noskill — APPROVED** [review]

**Review by noskill — APPROVED** [review]

**Pull request merged**